apps/haddock3-download/package.json

```diff
@@ -8,6 +8,7 @@
     "serve": "vite preview",
     "lint": "ts-standard",
     "test:integration": "playwright test",
+    "test:headed": "playwright test --project=chromium --headed",
```

Comment:
At https://github.com/i-VRESSE/workflow-builder/blob/130-start-list-with-1/README.md?plain=1#L80 the `--headed` option is mentioned. So this line feels like duplication.  Could you make them in sync with each other by either removing this line or in readme mention  `yarn test:headed`.